autoload/maktaba/path.vim

```diff
@@ -55,14 +84,13 @@ endfunction
 " The root of a relative path is empty.
 function! maktaba#path#RootComponent(path) abort
   if !s:is_backslash_platform
-    " This regex matches / alone.
-    return matchstr(a:path, s:root_frontslash)
+    return a:path[:0] is# '/' ? '/' : ''
```

Comment:
I lean towards `is#` by default because it's what == should have been, but either's fine here. Switched (here and elsewhere).